airflow/providers/amazon/aws/hooks/batch_client.py

```diff
@@ -36,7 +36,15 @@
 
 from airflow.exceptions import AirflowException
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
-from airflow.typing_compat import Protocol, runtime_checkable
```

Comment:
Not a huge fan of this.

Could we do this in the released versions only with bowler?

Reply:
Yeah I do not like it either. Let me see if I can do something about it .

Reply:
I think I solved all of those small quirks in a good way. The only "hacky" think is Parpermill AUTO lineage, because AUTO lineage is basically incompatible with Airflow 1.10. But the manual lineage should work.